sfn-query-large-datasets-cdk/README.md

```diff
@@ -0,0 +1,110 @@
+# Query large datasets (Amazon Athena, Amazon S3, AWS Glue, Amazon SNS)
+
+**Resources used**  
+
+| Resource Type | Name    |
+|------------------|--------------------|
+| Athena Workgroup | AthenaWorkgroup    |
+| State Machine    | AthenaStateMachine |
+| SNS Topic        | SNSTopic           |
+| S3 Bucket        | DataBucket         |
+| Glue Database    | GlueDatabase       |
+| Glue Crawler     | GlueCrawler        |  
+  
+## Start an Athena query
+
+This workflow demonstrates how to ingest a large data set in Amazon S3 and partition it through AWS Glue Crawlers, then execute Amazon Athena queries against that partition. Deploying this sample project creates an AWS Step Functions state machine, an Amazon S3 Bucket, an AWS Glue crawler, and an Amazon SNS topic.
+
+In this project, the Step Functions state machine invokes an AWS Glue crawler that partitions a large dataset in Amazon S3. Once the AWS Glue crawler returns a success message, the workflow executes Athena queries against that partition. Once query execution is successfully complete, an Amazon SNS notification is sent to an Amazon SNS topic.
+
+* An Amazon Athena query
+* An AWS Glue crawler
+* AWS Lambda Functions
+* An Amazon Simple Notification Service topic
+* Related AWS Identity and Access Management (IAM) roles
+
+Learn more about this workflow at Step Functions workflows collection: https://docs.aws.amazon.com/step-functions/latest/dg/sample-query-large-datasets.html
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. If this is your first time using AWS CDK, bootstrap your [environment](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_bootstrap).
+    ```
```

Comment:
Will adding the language to the markdown code blocks improved readiblity? 

See https://www.markdownguide.org/extended-syntax/#syntax-highlighting. Nevertheless, I am using it in my next comment as an example.